Date: 2024-09-21
Author: Bernhard M. Wiedemann <bwiedemann suse de>

Skip module signing for https://bugzilla.opensuse.org/show_bug.cgi?id=1217690
reproducible builds, until we can get a better upstream solution
such as
https://lists.libcamera.org/pipermail/libcamera-devel/2024-January/040244.html

diff --git a/src/meson.build b/src/meson.build
index 8eb8f05..41e556d 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -16,7 +16,7 @@ summary({
 
 # Module Signing
 openssl = find_program('openssl', required : false)
-if openssl.found()
+if false
     ipa_priv_key = custom_target('ipa-priv-key',
                                  output : ['ipa-priv-key.pem'],
                                  command : [gen_ipa_priv_key, '@OUTPUT@'])
